grails-app/controllers/org/pih/warehouse/api/ProductApiController.groovy

```diff
@@ -273,6 +275,14 @@ class ProductApiController extends BaseDomainApiController {
         render([data: catalogs] as JSON)
     }
 
+    def productGroupOptions = {
+        def productGroups = ProductGroup.list(sort: "name").collect {
+            [id: it.id, label: "${it.name}"]
+        }
+
+        render([data: productGroups] as JSON)
+    }
+
     def categoryOptions = {
```

Comment:
I think there is a new controller `SelectOptionsApiController` for this kind of methods.

Reply:
Thanks, I totally forgot about that

Reply:
I have also moved some of the other _get select options_ methods from `ProductApiController` to `SelectOptionsApiController` like tags, catalogs and categories